### DISCUSSION TITLE
聊天群招募新人

### DISCUSSION THREAD

**Discussion by bxzwq** [share121/inter-knot#5782] [公告]
![小三月](https://github.com/user-attachments/assets/b4399f6f-49b9-4fde-85c4-febfed97f573)

![芙芙](https://github.com/user-attachments/assets/c2748049-61af-4dda-a53f-60ff0ada4648)
![Screenshot_20241217_185505](https://github.com/user-attachments/assets/c40daced-4b5b-411b-add9-8b014b991fc9)

 现在的群人很少，目前来说就是啥都木有的群
问:群里有攻略吗？
答:没有(ノへ￣、)群主也需要攻略
问:群里有大佬吗？
答:好像也没有，群主也想要抱大佬大腿_(:з」∠)_不然就不会在这里发帖子招人了
（群里啥都木有还在宣传的群主是个屑）
还是有可以宣传的地方的，群相册有邦布可爱录像和谛听可爱录像，还有涩图（bing mei you）
还要写点什么来着，欢迎各位萌新大佬进群来玩909457307

**Comment by ChaoKaiADG** [comment]
开庭时带上你的三月七<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
